templates/cloudflare/package.json

```diff
@@ -16,12 +16,11 @@
     "@remix-run/cloudflare-pages": "*",
     "@remix-run/react": "*",
     "isbot": "^4.1.0",
-    "miniflare": "^3.20240404.0",
```

Comment:
@pcattori was there a reason miniflare was included? It doesn't seem like it was doing anything, and removing it didn't break `dev`, `start`, or `prod`

Reply:
🤷 don't remember, there was a lot of churn on the CF side when we were iterating towards the CF Vite stuff with them

Reply:
Cool, I'll just remove it since it worked fine without it locally for me

If I did something wrong people will yell at me and I can ask them to make our Cloudflare templates better. I mostly just want them to consistently work and be pretty minimal at the moment